### DISCUSSION TITLE
posts/proxmox

### DISCUSSION THREAD

**Discussion by giscus** [whexy/whexy-blog-comments#24] [General]
# posts/proxmox

介绍我的主机多操作系统解决方案：裸机虚拟化。

https://www.whexy.com/posts/proxmox

**Comment by TravisRoad** [comment]
最近捡了点洋垃圾，也装了pve玩。
问问博主，使用pve中的 windows 基本无感知在虚拟机当中吗，玩游戏是否有卡顿呢？😌

**Reply by whexy** [reply]
我已经用这套虚拟化的方案玩游戏差不多两个月了，实际效果比料想的好上不少（可能是之前我对虚拟化有偏见）。我举两个例子，一个是3A，一个是FPS网游。
- 最近在玩的《博德之门3》，比较吃 CPU 的游戏，帧数对比油管上同配置机器测试结果损耗大概在5%，基本没影响。
- 《守望先锋2》的体验也不错，但是在游戏过程中会有偶发的帧数暴跌，从200+跌到个位数，过几秒再涨回去。大概每三局就会出现一次。暂时还没有找出原因，猜测是pve在后台做某种调度导致的。

还有一个麻烦的事：如果游戏的反作弊策略比较激进，可能不允许在虚拟化环境里运行。比如Riot出品的《瓦洛兰特》，会直接提示不能启动。不过目前我只遇到这一款游戏有这种情况。

至于更吃性能的其他 3A 大作，我基本上用主机玩，毕竟 pve 上挂着一些不想暂停的服务，多多少少还是会占掉资源。

**Comment by seng1e** [comment]
半年多过去了，博主在虚拟化的windows中玩游戏还有遇到问题吗？

**Reply by whexy** [reply]
除了一些在线多人网络游戏因为严格的反作弊机制不让用虚拟机运行，没遇到过别的什么问题

**Comment by johnnyasantoss** [comment]
Great read
